packages/framework/oldest-client-observer/package.json

```diff
@@ -85,7 +85,7 @@
 		"typescript": "~4.5.5"
 	},
 	"typeValidation": {
-		"broken": {},
-		"disabled": true
+		"disabled": true,
+		"broken": {}
```

Comment:
Nothing major, but maybe we want the code that rewrites this to always output the keys alphabetically?

Reply:
It should be deterministic order, but I did expect it to be alphabetical. I'll follow up.